### PR TITLE
Update cookbooks for openfe 1.0

### LINKED ISSUE #118: Choose and Configure a Protocol cookbook is out of date
There are a couple aspects to this Protocol which are out of date for the 1.0 release.

### PR DESCRIPTION
**Pull request by IAlibay**
Fixes #118 

Fixes coobooks for 1.0rc

### CODE REVIEW

File: cookbook/choose_protocol.ipynb [hunk 1]
```diff
@@ -4,7 +4,6 @@
    "cell_type": "markdown",
```
Comment (hannahbaumann) [inline cookbook/choose_protocol.ipynb:4]: <pre class="ql-syntax" spellcheck="false">Line #33.            off_toolkit_backend='ambertools',  # Toolkit to use for partial charge assignment - ambertools
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenFreeEnergy/ExampleNotebooks/pull/122/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Comment (hannahbaumann) [inline cookbook/choose_protocol.ipynb:4]: <pre class="ql-syntax" spellcheck="false">Line #51.            # False: don't apply excplicit charge correction using an alchemical water
Line #51.            # False: don't apply explicit charge correction using an alchemical water
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenFreeEnergy/ExampleNotebooks/pull/122/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

**Review by hannahbaumann — COMMENTED** [review]
Two small spelling things, otherwise lgtm! I haven't tried and run it myself, please let me know if that would be helpful!

**Comment by IAlibay** [comment]
>  I haven't tried and run it myself, please let me know if that would be helpful!

I don't think it's necessary, but if you feel like it more checks doesn't hurt either.

**Review by hannahbaumann — APPROVED** [review]

**Pull request merged**